src/features/surveys/components/surveyForm/SignUpOptions.tsx

```diff
@@ -0,0 +1,33 @@
+import messageIds from 'features/surveys/l10n/messageIds';
+import { useMessages } from 'core/i18n';
+import { FormControlLabel, Radio, RadioGroup } from '@mui/material';
+
+interface SignUpOptionsProps {
+  signature:
+    | 'require_signature'
+    | 'allow_anonymous'
+    | 'force_anonymous'
+    | undefined;
+}
+const SignUpOptions = ({ signature }: SignUpOptionsProps) => {
```

Comment:
I don't think this is used anywhere. Is that right?

Reply:
Correct, and deleted in 933c9dcc6d589f6905911cbcb9499a7a1de983cf!